### PR TITLE
Added ReaktorConfigure Annotation

### PR DESCRIPTION
**Pull request by dpwspoon**
Useful for json where the nukleus references

```java
    @Test
    @ReaktorConfigure(
        name="TLS_Keystore"
        value = "/Users/Jane/keystore.db")
    @Specification({
        "${route}/proxy/controller",
        "${streams}/append.request.header/accept/client",
        "${streams}/append.request.header/connect/server",
        })
    public void shouldAppendHeaderOnRequest() throws Exception
    {
        k3po.finish();
    }
```

### CODE REVIEW

File: src/test/java/org/reaktivity/reaktor/test/annotation/Configure.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ ElementType.METHOD })
+@Repeatable(Configures.class)
+public @interface Configure
```
Comment (jfallows) [inline src/test/java/org/reaktivity/reaktor/test/annotation/Configure.java]: Rename to `ReaktorConfiguration` ?

File: src/test/java/org/reaktivity/reaktor/test/annotation/Configures.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+package org.reaktivity.reaktor.test.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ ElementType.METHOD })
+public @interface Configures
```
Comment (jfallows) [inline src/test/java/org/reaktivity/reaktor/test/annotation/Configures.java]: Rename to `ReaktorConfigurations` ?

File: src/test/java/org/reaktivity/reaktor/test/annotation/Configure.java (hunk 1 above)
Comment (dpwspoon) [reply src/test/java/org/reaktivity/reaktor/test/annotation/Configure.java]: The configure matches the `ReaktorRule#configure`, how about ReaktorConfigure and ReaktorConfigures for the annotations?

Or, if we change it to ReaktorConfiguration, should we change the builder to be Configuration.

Comment (jfallows) [reply src/test/java/org/reaktivity/reaktor/test/annotation/Configure.java]: `ReaktorConfigure` and `ReaktorConfigures` works!

**Pull request merged**